README.md

```diff
@@ -13,43 +13,54 @@
 
 ## Features
 
-- Support multiple experiments (AB or MVT[Multi-Variant])
+- Supports multiple experiments (AB or MVT[Multi-Variant])
 - Auto assign experiment/variant to users
 - SSR support using cookies
 - CSS and state injection
-- Automatically revoke expired experiments from testers
+- Automatically revoke expired experiments and assign new ones
 - Ability to assign experiments based on context conditions (Route, State, etc)
+- Works well with Google Tag Manager and Google Analytics
 
 ## Setup
 
-- Add `nuxt-google-optimize` dependency using yarn or npm to your project
+Add `nuxt-google-optimize` dependency using yarn or npm to your project.
 ```sh
+# using yarn
 yarn add nuxt-google-optimize
-```
-OR
-```sh
+# using npm
 npm install nuxt-google-optimize --save
 ```
 
-- Add `nuxt-google-optimize` to `modules` section of `nuxt.config.js`
+Add `nuxt-google-optimize` to `modules` section of `nuxt.config.js`
 
 ```js
 {
   modules: [
     'nuxt-google-optimize',
   ],
 
-  // Optional options
+  // Options
   googleOptimize: {
     // experimentsDir: '~/experiments',
-    // maxAge: 60 * 60 * 24 * 7 // 1 Week
+    // maxAge: 604800 // 1 week (in seconds)
```

Comment:
`60 * 60 * 24 * 7` convention is to allow developers quickly modify time without need to calculations and increasing source readability